example/lint.sh

```diff
@@ -25,6 +25,7 @@ filter='.namedSetOfFiles | values | .files[] | ((.pathPrefix | join("/")) + "/"
 # NB: perhaps --remote_download_toplevel is needed as well with remote execution?
 args=(
 	"--aspects=$(echo //tools/lint:linters.bzl%{buf,eslint,flake8,ktlint,pmd,ruff,shellcheck,vale} | tr ' ' ',')"
+	"--norun_validations"
```

Comment:
Perhaps common this flag with a link to https://github.com/aspect-build/rules_ts/pull/574#issuecomment-2073632879 so future reads won't have to use blame to find out why it is here?